### PR TITLE
Minor changes to fix switching of resolutions.

### PR DESCRIPTION
**Pull request by Langerz82**
A more cleaner way of switching Resolutions, so screen corruption does not occur.

### CODE REVIEW

File: packages/sx05re/tools/sysutils/gptokeyb/package.mk [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Copyright (C) 2021-present Shanti Gilbert (https://github.com/shantigilbert)
 
 PKG_NAME="gptokeyb"
-PKG_VERSION="928dc845ee665528fd510a75be724eb991ebb6f1"
+PKG_VERSION="ded03731ccf8e65e855dd776aade021425afc53b"
```
Comment (Langerz82) [inline packages/sx05re/tools/sysutils/gptokeyb/package.mk:5]: Update gptokeyb package build.

**Comment by Langerz82** [comment]
In the former code, when an error occurs and TvTextViewer is shown the display can corrupt. This code is much cleaner. It blanks the buffers and restores only when the Res is fully switched, and no need to call the show splash twice.

**Review by shantigilbert — CHANGES_REQUESTED** [review]

File: packages/sx05re/tools/sysutils/gptokeyb/package.mk (hunk 1 above)
Comment (Langerz82) [reply packages/sx05re/tools/sysutils/gptokeyb/package.mk]: Done. But yeah please update the gptokeyb version please!

**Comment by shantigilbert** [comment]
Thank you!

**Pull request merged**